python/cuml/cuml/accel/tests/scikit-learn/xfail-list.yaml

```diff
@@ -502,10 +502,6 @@
   - "sklearn.inspection.tests.test_partial_dependence::test_multiclass_multioutput[RandomForestClassifier]"
   - "sklearn.inspection.tests.test_partial_dependence::test_partial_dependence_sample_weight_with_recursion"
   - "sklearn.inspection.tests.test_partial_dependence::test_recursion_decision_tree_vs_forest_and_gbdt[0]"
-  - "sklearn.inspection.tests.test_permutation_importance::test_permutation_importance_correlated_feature_regression[ones-0.5-1]"
-  - "sklearn.inspection.tests.test_permutation_importance::test_permutation_importance_correlated_feature_regression[ones-0.5-2]"
-  - "sklearn.inspection.tests.test_permutation_importance::test_permutation_importance_correlated_feature_regression[ones-1.0-1]"
-  - "sklearn.inspection.tests.test_permutation_importance::test_permutation_importance_correlated_feature_regression[ones-1.0-2]"
```

Comment:
For some reason, `sklearn.inspection.tests.test_permutation_importance::test_permutation_importance_correlated_feature_regression` no longer fails with `cuml.accel`. I've removed them from `xfail-list.yaml`. Do we happen to have an issue open for this test suite?

Reply:
Rough guess: the new FIL uses float64 to match the native precision for sklearn RF models, so it produces more accurate output than the legacy FIL (which uses float32).